src/audio/audio.hh

```diff
@@ -0,0 +1,13 @@
+#ifndef VOCODER_AUDIO_HH
+#define VOCODER_AUDIO_HH
+
+class Audio {
+  public:
+    Audio(std::vector<float> sample_) : sample(sample_) {
+    }
+
+  private:
+    std::vector<float> sample;
+};

```

Comment:
If the data is prviate and there is no getter/setter, then this data type can't be accessed

Reply:
changed to a struct since it is only encapsulating data.